toolchain/language_server/language_server.cpp

```diff
@@ -12,14 +12,17 @@
 
 namespace Carbon::LanguageServer {
 
-auto Run(std::FILE* input_stream, llvm::raw_ostream& output_stream)
-    -> ErrorOr<Success> {
+auto Run(std::FILE* input_stream, llvm::raw_ostream* output_stream,
+         llvm::raw_ostream* error_stream) -> ErrorOr<Success> {
+  CARBON_CHECK(input_stream && output_stream && error_stream);
```

Comment:
If input_stream and output_stream have to be non-null, could they be passed by reference here instead of by pointer?

Reply:
Oh, wait I guess these can be references.